.github/workflows/pytest-run.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
Same here.

Reply:
I like having the tests on PRs run only for PRs into master, but again it is a personal preference and git usage question. Feel free to remove this or use '*' to indicate all branches. Here is the relevant docs from github: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on